basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/LTIRoleMapperImpl.java

```diff
@@ -106,25 +104,21 @@ public void setSiteService(SiteService siteService) {
                     M_log.debug("No match, falling back to determine role");
                 }
 
-                String maintainRole = site.getMaintainRole();
-                String joinerRole = site.getJoinerRole();
+                ServerConfigurationService cnf = (ServerConfigurationService) ComponentManager
+                		.get(ServerConfigurationService.class);
+				String maintainRole = site.getMaintainRole();
```

Comment:
Since the site service is injected the server configuration service should be injected too.  This is a simple update to components.xml.  It ends up really confusing to use two difference approaches to getting dependencies.